docs/src/tutorials/linear/multi_objective_examples.jl

```diff
@@ -30,11 +30,7 @@ set_silent(model)
 @objective(model, Min, [3x1 + x2, -x1 - 2x2])
 @constraint(model, 3x1 - x2 <= 6)
 set_optimizer(model, () -> MOA.Optimizer(HiGHS.Optimizer))
-set_attribute(
```

Comment:
Intended diff?

Reply:
Yip. I just noticed when I checked the versions we were using. Didn't seem worth a separate PR when I was doing this one.